base.Makefile

```diff
@@ -324,6 +324,8 @@ clea%: least-specific-clean ## clean: Clean all built assets
 lint-pytho%: ## lint-python: Lint python source
 	$(LOG)
 	$(RUFF) check $(LINTED_PYTHON_DIRS)
+	$(DJLINT) $(TEMPLATE_DIRS) --lint
```

Comment:
Please add a `TEMPLATE_DIRS` variable default value into `## Python env` block in this file. Something like: 

`TEMPLATE_DIRS ?= $(PYTHON_SRCDIR)/templates`

Reply:
Please check that it does not break.

Reply:
Done :+1: